docs/rfcs/1013-parsec-account.md

```diff
@@ -264,6 +264,10 @@ Attributes:
 >   still be used to recover the vault in case of account recovery (see below).
 > - Currently, only password authentication is supported, other authentication
 >   methods (such as FIDO2) may be added in the future.
+> - Given the server control the password algorithm, the client must ensure the
+>   provided configuration is not too weak (which would allow the server to brute
+>   force the password by using `auth_method_id` as oracle).
```

Comment:
What should be the expected client behavior if the configuration is too weak ? Just saying that the configuration is too weak is not really helpful, and there is nothing a user could really do to solve that. Maybe something like "contact an administrator " ?

Reply:
:+1: